client/utils/account-fees.js

```diff
@@ -5,24 +5,172 @@
  */
 import { __, sprintf } from '@wordpress/i18n';
 import { __experimentalCreateInterpolateElement as createInterpolateElement } from 'wordpress-element';
+import interpolateComponents from 'interpolate-components';
+import './account-fees.scss';
 
 /**
  * Internal dependencies
  */
 import { formatCurrency } from 'utils/currency';
 import { formatFee } from 'utils/fees';
 
+const countryFeeStripeDocsBaseLink =
+	'https://woocommerce.com/document/payments/faq/fees/#section-';
+const countryFeeStripeDocsSectionNumbers = {
+	AU: 1,
+	AT: 2,
+	BE: 3,
+	CA: 4,
+	FR: 5,
+	DE: 6,
+	HK: 7,
+	IE: 8,
+	IT: 9,
+	NL: 10,
+	NZ: 11,
+	PL: 12,
+	PT: 13,
+	SG: 14,
+	ES: 15,
+	CH: 16,
+	UK: 17,
+	US: 18,
+};
+
+const getStripeFeeSectionUrl = ( country ) => {
```

Comment:
Should we add logic to return a generic link to the page if the country isn't in our `countryFeeStripeDocsSectionNumbers` list? Just thinking we may forget to update this if we launch in new countries 😅

Reply:
Makes sense. I'll add a generic link directing the merchant to https://woocommerce.com/document/payments/faq/fees/ if the country doesn't exist in that list. :+1:

Reply:
Solved in 94ede4f. Thanks for the suggestion :)